src/EditorFeatures/TestUtilities/CodeActions/CSharpCodeFixVerifier`2.cs

```diff
@@ -72,5 +74,48 @@ public static async Task VerifyCodeFixAsync(string source, DiagnosticResult[] ex
             test.ExpectedDiagnostics.AddRange(expected);
             await test.RunAsync();
         }
+
+        public static async Task VerifyCodeFixAsync((string first, string second) sources, (string first, string second) fixedSources, int numberOfFixAllIterations)
+            => await VerifyCodeFixAsync(sources, fixedSources, DiagnosticResult.EmptyDiagnosticResults, numberOfFixAllIterations);
+
+        public static async Task VerifyCodeFixAsync((string first, string second) sources, (string first, string second) fixedSources, DiagnosticResult[] expected, int numberOfFixAllIterations)
+        {
+            var test = new Test
+            {
+                TestState = { Sources = { sources.first, sources.second } },
+                FixedState = { Sources = { fixedSources.first, fixedSources.second } },
+                NumberOfFixAllInDocumentIterations = numberOfFixAllIterations
+            };
+
+            test.ExpectedDiagnostics.AddRange(expected);
+            await test.RunAsync();
+        }
+
+        /// <summary>
+        /// Verifies fix for a single diagnostic and iterative batch fix for all diagnostics.
+        /// </summary>
+        /// <param name="source">Original source with multiple diagnostics.</param>
+        /// <param name="fixedSource">Fixed source with code fix applied for a single diagnostic selected by <paramref name="diagnosticSelector"/>.</param>
+        /// <param name="batchFixedSource">Fixed source for iterative batch fix for all diagnostics.</param>
```

Comment:
📝 This is used for all four fix approaches (iterative, fix all in document/project/solution).

Reply:
Will adjust the doc comment.